templates/cosmic-settings.tera

```diff
@@ -0,0 +1,135 @@
+---
+name: "Catppuccin"
+roundness: "round"
+window_hint_color: ""
+bg_alpha: 1.0
+frosted: false
+outer_gap_size: 0
+inner_gap_size: 8
+active_hint_size: 3
+whiskers:
+    version: 2.5.1
+    matrix:
+    - flavor
+    - accent
+    filename: "themes/{{ name }}-{{ flavor.identifier | capitalize }}-{{ accent | capitalize }}.ron"
```

Comment:
> I imagine a more common use of it might be for a single user to have a theme which only overrides i.e. the roundness, and name it CatppuccinSquare or something just to be distinguishable from the main theme.

It looks like what we want to follow is our (now archived) [GTK theme zip naming convention](https://github.com/catppuccin/gtk/releases/tag/v1.0.3) where we had a bunch of customisation options but defaulted to some. If the roundness is common enough to be overidden then we should have it like so:

```suggestion
    filename: "themes/cosmic-settings/catppuccin-{{ flavor.identifier }}-{{ accent }}+{{ roundness }}.ron"
```

Reply:
That makes sense. AFAIK, the roundness is sort of orthogonal to the palette (which is the main concern of catppuccin), but we should have a way to control it anyway, since there's not a way to set just the colors without also setting the roundness. In that case, I could put it back to being used as the "variant" like it was before, so it's generated in the matrix with the flavor and accent, but that's a *lot* of theme variants to be generating (168, unless we want to reduce the number of generated accent colors.) What do you think?

Reply:
I'm not so keen on having it as part of the matrix to put 168 files into source control and I wouldn't want to reduce the number of accent generated files either.

We didn't generate the full matrix for GTK assets so I think it's fine putting `{{ roundness }}` into the filename by default. The README has information on how to override the options too which is good.